parts/k8s/containerdtemplate.toml

```diff
@@ -32,6 +32,7 @@ state = "C:\\ProgramData\\containerd\\state"
     max_container_log_line_size = 16384
     [plugins.cri.containerd]
       snapshotter = "windows"
+      discard_unpacked_layers = true
```

Comment:
Is this setting related to logging?

Reply:
no. was given guidance to use this setting in k8s env which will help with disk space. I could submit it as separate pr if we want.

Reply:
nah, just curious